pkg/console/subresource/deployment/deployment.go

```diff
@@ -229,20 +229,26 @@ func consoleVolumeMounts(vc []volumeConfig) []corev1.VolumeMount {
 	return volMountList
 }
 
-type LogMapping map[operatorv1.LogLevel]string
-
-var LogMap = LogMapping{
-	operatorv1.Normal:   "--log-level=*=NOTICE",
-	operatorv1.Debug:    "--log-level=*=DEBUG",
-	operatorv1.Trace:    "--log-level=*=TRACE",
-	operatorv1.TraceAll: "--log-level=*=TRACE",
+func GetLogLevelFlag(logLevel operatorv1.LogLevel) string {
+	flag := ""
+	switch logLevel {
+	case operatorv1.Normal:
+		flag = "--log-level=*=NOTICE"
+	case operatorv1.Debug:
+		flag = "--log-level=*=DEBUG"
+	case operatorv1.Trace:
```

Comment:
Would it be helpful to reuturn both `(string, ok)`?  
Instead of `fallthrough`, use double matching case (this seems to be the pattern I see more often in our codebases):

```golang
// do we validate loglevel on admission? if so, `ok` isn't needed. otherwise could be helpful 
func GetLogLevelFlag(brandstr string) (logLevel string, ok bool) {
   flag := ""
   switch logLevel {
   case operatorv1.Normal:
      flag = "--log-level=*=NOTICE"
   case operatorv1.Debug:
      flag = "--log-level=*=DEBUG"
   // i think this is the preferred idiom
   case operatorv1.Trace, operatorv1.TraceAll:
      flag = "--log-level=*=TRACE"
   }
   return flag
}
```
Though I do not remember offhand if we validate `LogLevel` at the config level. If so, its not possible to get an invalid value, in which case `found` is unnecessary.